testing/dbrp_mapping.go

```diff
@@ -989,6 +991,19 @@ func FindDBRPMappingByIDV2(
 		{
 			name: "find non existing dbrp",
 			fields: DBRPMappingFields{
+				BucketSvc: &mock.BucketService{
```

Comment:
I may be reading these tests wrong, but they look like they're all negative tests around expecting the dbrp not to exist. If that's the case, it would be nice to see a some positive test cases, particularly around the logic for handling the buckets with slashes

Reply:
That's a good point, I'll add some.